### ISSUE TITLE
Allow verbose/non-silent logging for pkg5 module

### ISSUE THREAD

**Issue by Klistel** [ansible-collections/community.general#8379]
### Summary

I'm using the pkg5 module to update a number of solaris 11 servers and I'm unable to find any kind of logging flag as part of the module. The module seems to run pkg in -q mode, but my sysadmin team is adamant that they have access to the verbose output of the pkg command. Is there a way to enable normal logging for capture by "register"?

Current output:

   "results": [
      "\nA clone of SRU-11.4.45.0.1 exists and has been updated and activated.\nOn the next boot the Boot Environment be://rpool/11.4.68.0.1.164.2 will be\nmounted on '/'.  Reboot when ready to switch to this updated BE.\n\n"
    ],

Desired output:

 Startup: Linked image publisher check ... Done
 Startup: Refreshing catalog 'solaris' ... Done
 Startup: Refreshing catalog 'solarisstudio' ... Done
Planning: Solver setup ... Done
Planning: Running solver ... Done
Planning: Finding local manifests ... Done
... lots more output ....
Finalize: Finished processing linked images.
 Done

A clone of S11.4-SRU65.0.1.157.1 exists and has been updated and activated.
On the next boot the Boot Environment be://rpool/11.4.68.0.1.164.2 will be
mounted on '/'.  Reboot when ready to switch to this updated BE.

 Done
<EOF>

### Issue Type

Feature Idea

### Component Name

pkg5

### Additional Information

```- name: Update pkg and refresh publisher
  community.general.pkg5:
    name: "{{ pkgName }}"
    be_name: "{{ patching_sruversion }}"
    refresh: true
    accept_licenses: true
    verbose: true
  register: pkg5_command

```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by ansibullbot** [comment]
Files identified in the description:
* [`plugins/modules/pkg5.py`](https://github.com/ansible-collections/community.general/blob/main/plugins/modules/pkg5.py)

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/devel/ISSUE_HELP.md)

**Comment by ansibullbot** [comment]
cc @bcoca @fishman @jasperla @jpdasma @mator @mavit @scathatheworm @troy2914 @xen0l
[click here for bot help](https://github.com/ansible/ansibullbot/blob/devel/ISSUE_HELP.md)

**Comment by Klistel** [comment]
I am not 100% sure how this works as I've never contributed to a project on here before, but I looked into the code and it seemed like it was a fairly simple add to allow the user to turn off the hardcoded "-q" flag in the command. I used the existing params/arguments in the module as a baseline for what I attempted.

https://github.com/ansible-collections/community.general/commit/d626148e685c9c2034d7edc622262289890b5488

I don't know if I'm going about this the right way so forgive me if I'm overstepping on this.
